_reports/shrikrishna.md

```diff
@@ -0,0 +1,255 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+date: 2018-08-12
+student: Shrikrishna Singh
+organisation: coala
+organisation_link: https://coala.io
+report_year: 2018
+project: Newcomer Metrics and Gamification
+project_link: https://summerofcode.withgoogle.com/projects/#5892040252981248
+tarball: https://github.com/sks444/GSoC-18-Patches/raw/master/project.tar.gz
+sha256sum: 43e4291bf4a8b6d83b004483d260cf2dfdd0d3ae139d598f40e473d05513f9c6
+mentors: >
+  [John Vandenberg](http://github.com/jayvdb),
+  [Pranav Dixit](https://github.com/prnvdixit),
+  [Andrew Dassonville](https://github.com/andrewda) &
+  [Gopala](https://github.com/gkrishnan724)
+phase:
+  - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/2
+  - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/19
+  - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/43
+  - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/44
+bio: >
+  I am a final year Computer Science and Engineering undergraduate at
+  Shri Mata Vaishno Devi University. As part of the GSoC'18 I worked on
+  [community](https://github.com/coala/community) and
+  [webservices](https://gitlab.com/coala/landing/) project to build a
+  gamification system for all the active newcomers at coala.
+
+social:
+  - GitHub:
+    - username: sks444
+    - link: https://github.com/sks444
+  - GitLab:
+    - username: sks444
+    - link: https://gitlab.com/sks444
+  - Gitter:
+    - username: sks444
+    - link: https://gitter.im/sks444
+  - OpenHub:
+    - username: sks444
+    - link: https://www.openhub.net/accounts/sks444
+email: krishnasingh.ss30@gmail.com
+blog: https://sks444.github.io
+activity:
+  - 1:
+    - repo: coala-bears
+    - link: https://github.com/coala/coala-bears/commit/d35238fd
+    - details: >
+       GitCommitBear: Parse remote with giturlparse
+  - 2:
+    - repo: coala
+    - link: https://github.com/coala/coala/commit/8dfe8376
+    - details: >
+       Jinja2.py: Add support for ignore comments
+  - 3:
+    - repo: cEPs
+    - link: https://github.com/coala/cEPs/commit/1ec5b92c
+    - details: >
+       cEP-0020: Add Newcomer metrics and Gamification system
+  - 4:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/77f8802e
+    - details: >
+       org/cron.py: Use dict.get method to r_json
+  - 5:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/791baab0
+    - details: >
+       Add IGitt Cache mechanism
+  - 6:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/1b8058c0
+    - details: >
+       requirements.txt: Use == for installing IGitt
+  - 7:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/00f3081e
+    - details: >
+       Serialize coala issues on GitHub and GitLab
+  - 8:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/1398c931
+      details: >
+       Implement logging
```

Comment:
@jayvdb, why this description is not showing up in https://deploy-preview-619--coala-projects.netlify.com/gsoc/2018/statusreport/shrikrishna ?

Reply:
it needs `-`